app/graphql/types/mutation_type.rb

```diff
@@ -45,6 +55,50 @@
     resolve Mutations::DeleteOrganizationMutation.new
   end
 
+  field :resendConfirmationEmail, !types.Boolean do
+    description 'Resends the confirmation email to a user'
+
+    argument :email, !types.String do
+      description 'The email address of the user'
+    end
+
+    resolve Mutations::ResendConfirmationEmailMutation.new
+  end
+
+  field :resendPasswordResetEmail, !types.Boolean do
+    description 'Resends the password reset email to a user'
+
+    argument :email, !types.String do
+      description 'The email address of the user'
+    end
+
+    resolve Mutations::ResendPasswordResetEmailMutation.new
+  end
+
+  field :resendUnlockAccountEmail, !types.Boolean do
+    description 'Resends the unlock account email to a user'
+
+    argument :email, !types.String do
+      description 'The email address of the user'
+    end
+
+    resolve Mutations::ResendUnlockAccountEmailMutation.new
+  end
+
+  field :resetPassword, !Types::SessionTokenType do
```

Comment:
I belive this must be of type `Types::SessionTokenType` (without the bang). Background for this is that if an error is set on a field, this field will be set to `null`. If this happens on a field that can't be `null`, the parent field will be set to `null` and so on. Mutations on the other hand are always top-level fields, so if an mutation can raise an error, we must always omit the `!`. Otherwise we would get another error reporting that the field can't be `null`.